src/Files.App/BaseLayout.cs

```diff
@@ -606,6 +600,24 @@ private async Task LoadMenuItemsAsync()
 			}
 		}
 
+		private void AddCloseHandler(IList<ICommandBarElement> primaryElements, IList<ICommandBarElement> secondaryElements)
+		{
+			// Workaround for WinUI (#5508)
```

Comment:
Can you include a link to the issue?

Reply:
The comment comes from a previous fix for primary elements. This pr fixes the submenus of the secondary elements. Is it worth keeping this comment? What link should I put?

Reply:
If it's still a workaround, we should link to the issue on WinUI